src/lib/IFrameCommunicator.ts

```diff
@@ -42,10 +26,12 @@ export interface IFrameMessage<E extends keyof IFrameMessagesMap> {
   readonly type: E;
   readonly data: Parameters<IFrameMessagesMap[E]>[0];
   readonly metadata?: Parameters<IFrameMessagesMap[E]>[1];
+  readonly operationId?: string;
 }
 
 export class IFrameCommunicator {
   private readonly events: EventManager<IFrameMessagesMap>;
+  private readonly callbacks: any = {};
```

Comment:
Please change this `any` with a type e.g. Map<string, Callback> or Record<string, Callback>.